src/test/java/com/bae/cookbook/RecipeRestUnitTest.java

```diff
@@ -0,0 +1,115 @@
+package com.bae.cookbook;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.aspectj.lang.annotation.Before;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.bae.persistence.domain.Recipe;
+import com.bae.persistence.repository.RecipeRepository;
+import com.bae.rest.RecipeController;
+import com.bae.service.RecipeService;
+
+@RunWith(SpringRunner.class)

```

Comment:
Think this should be run with mockito as otherwise will try and run a full springboot app rather than mocking it
```suggestion
@RunWith(MockitoJUnitRunner.class)
```